packages/arb-token-bridge-ui/src/components/TransferPanel/EstimatedGas.tsx

```diff
@@ -119,7 +123,11 @@ export function EstimatedGas({
     return <GasFeeForClaimTxMessage networkName={parentChainName} />
   }
 
-  if (isTokenNativeUSDC(selectedToken?.address) && !isSourceChain) {
+  if (
+    isTokenNativeUSDC(selectedToken?.address) &&
+    !isSourceChain &&
+    !isDestinationChainOrbit
```

Comment:
```suggestion
    isTokenNativeUSDC(selectedToken?.address) &&
    !isSourceChain &&
    (isDestinationArbSepolia || isDestinationArbOne)
```

I think this is more accurate.... I also notice that we are currently showing this claim tx message for Ethereum -> Nova USDC deposit which is wrong as it is not done through CCTP (my bug sorry sorry)